Docker/build/WDQS/Dockerfile

```diff
@@ -30,13 +30,14 @@ ENV MEMORY=""\
     WDQS_ENTITY_NAMESPACES="120,122"\
     WIKIBASE_SCHEME="http"\
     WIKIBASE_MAX_DAYS_BACK="90"\
-    BLAZEGRAPH_EXTRA_OPTS=""
+    BLAZEGRAPH_EXTRA_OPTS=""\
+    LOG_CONFIG="/wdqs/logback.xml"
```

Comment:
The WDQS changes in this PR are covered in https://phabricator.wikimedia.org/T344549